README.md

```diff
@@ -2,6 +2,15 @@
 
 # Watson Stock Advisor
 
+![architecture](docs/architecture.png)
+
+## Flow
+1. The user interacts with the backend server via the app UI. Here they are able to add and remove stocks they are interested in.
+2. User input is processed and routed to the backend server, which in turn uses Watson Discovery and a Cloudant NoSQL database. 
+3. The backend server stores stock information in a Cloudant NoSQL database for easy retrieval.
+4. The backend server uses the Watson Discovery  to find information about a specific company.
```

Comment:
extra space between 'Discovery' and 'to'

Reply:
fixed